<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Attendence marking System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/attendance.png</normaloff>:/images/attendance.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>-3</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: lightgray;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcomeLable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>26</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 39, 51);</string>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Register New User</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 184, 230);
                                  color: white;
                                  font-size: 22px;
                                  border-style: solid;
                                  border-width: 2px;
                                  border-radius: 10px;
                                  }
                                  QPushButton:pressed {
                                  background-color:rgb(0, 143, 179);</string>
    </property>
    <property name="text">
     <string>Register User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rulesButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>See Our Rules</string>
    </property>
    <property name="toolTipDuration">
     <number>-3</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 184, 230);
                                  color: white;
                                  font-size: 22px;
                                  border-style: solid;
                                  border-width: 2px;
                                  border-radius: 10px;
                                  }
                                  QPushButton:pressed {
                                  background-color:rgb(0, 143, 179);

									</string>
    </property>
    <property name="text">
     <string>View Rules</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Generate and Download Report</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 184, 230);
                                  color: white;
                                  font-size: 22px;
                                  border-style: solid;
                                  border-width: 2px;
                                  border-radius: 10px;
                                  }
                                  QPushButton:pressed {
                                  background-color:rgb(0, 143, 179);</string>
    </property>
    <property name="text">
     <string>Download Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="markButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Mark your attendence</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 184, 230);
                                  color: white;
                                  font-size: 22px;
                                  border-style: solid;
                                  border-width: 2px;
                                  border-radius: 10px;
                                  }
                                  QPushButton:pressed {
                                  background-color:rgb(0, 143, 179);</string>
    </property>
    <property name="text">
     <string>Mark</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
